ttkwidgets/table.py

```diff
@@ -0,0 +1,481 @@
+                    if item != '':
+                        bbox = self.bbox(item)
+                        if not bbox:
+                            # the item is not visible so make it visible
+                            self.see(item)
+                            self.update_idletasks()
+                            bbox = self.bbox(item)
+                        if y < self._dragged_row_y - bbox[3] / 2:
+                            # the row is beyond half of item, so insert it above
+                            self._move_dragged_row(item)
+                        elif item != self.prev(self._dragged_row):
+                            # item is not the upper neighbor of the dragged row so insert the row below
+                            self._move_dragged_row(self.next(item))
+                self.selection_remove(self._dragged_row)
+
+    def _sort_column(self, column, reverse):
+        """Sort column."""
+        if 'disabled' not in self.state():
+            # get list of (value, item) tuple where value is the value in column for the item
+            l = [(self.set(child, column), child) for child in self.get_children('')]
+            # sort list using the column type
+            l.sort(reverse=reverse, key=lambda x: self._column_types[column](x[0]))
+            # reorder items
+            for index, (val, child) in enumerate(l):
+                self.move(child, "", index)
+            # reverse sorting direction for the next time
+            self.heading(column, command=lambda: self._sort_column(column, not reverse))
+
+    def cget(self, key):
+        if key == 'sortable':
+            return self._sortable
+        elif key == 'drag_cols':
+            return self._drag_cols
+        elif key == 'drag_rows':
+            return self._drag_rows
+        else:
+            return ttk.Treeview.cget(self, key)
+
+    def column(self, column, option=None, **kw):
+        """
+        Query or modify the options for the specified column.
+
+        If kw is not given, returns a dict of the column option values. If
+        option is specified then the value for that option is returned.
+        Otherwise, sets the options to the corresponding values.
+
+        options: 'type' (content type for sorting, default=str), all Treeview column options
+        """
+        config = False
+        if option == 'type':
+            return self._column_types[column]
+        elif 'type' in kw:
+            config = True
+            self._column_types[column] = kw.pop('type')
+        if kw:
+            self._visual_drag.column(ttk.Treeview.column(self, column, 'id'), option, **kw)
+        if kw or option:
+            return ttk.Treeview.column(self, column, option, **kw)
+        elif not config:
+            res = ttk.Treeview.column(self, column, option, **kw)
+            res['type'] = self._column_types[column]
+            return res
+
+    def configure(self, cnf=None, **kw):
+        if cnf == 'drag_cols':
```

Comment:
When I run `ttk.Button(text='Hello').configure(cnf='text')` I get  `('text', 'text', 'Text', '', 'Hello')`, but I admit that I never use the `configure` function that way.  So we don't have to keep this part if you don't want to.

By the way I noticed that I forgot to check if 'drag_cols', 'drag_rows' and 'sortable' were in `cnf` or not. But I guess it would be better to make the change in table_funcs.